UnitsNet.Tests/CustomCode/FluidResistanceTests.cs

```diff
@@ -0,0 +1,49 @@
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class FluidResistanceTests : FluidResistanceTestsBase
+    {
+        // https://www.vcalc.com/wiki/Fluid-Resistance-Unit-Conversion
+
+        protected override bool SupportsSIUnitSystem => false;
+        protected override double DyneSecondsPerCentimeterToTheFifthInOnePascalSecondPerCubicMeter => 1e5; // dyn·s/cm⁵
+        protected override double MegapascalSecondsPerCubicMeterInOnePascalSecondPerCubicMeter => 1e-6; // MPa·s/m³
+        protected override double MillimeterMercuryMinutesPerCubicCentimeterInOnePascalSecondPerCubicMeter => 1.25010625903202e-10; // mmHg·min/cm³
+        protected override double MillimeterMercuryMinutesPerCubicMeterInOnePascalSecondPerCubicMeter => 1.25010625903202e-4; // mmHg·min/m³
+        protected override double MillimeterMercuryMinutesPerLiterInOnePascalSecondPerCubicMeter => 1.25010625903202e-7; // mmHg·min/l
+        protected override double MillimeterMercuryMinutesPerMilliliterInOnePascalSecondPerCubicMeter => 1.25010625903202e-10; // mmHg·min/ml
+        protected override double MillimeterMercurySecondsPerCubicCentimeterInOnePascalSecondPerCubicMeter => 7.5006375541921e-9; // mmHg·s/cm³
+        protected override double MillimeterMercurySecondsPerCubicMeterInOnePascalSecondPerCubicMeter => 7.50063755419211e-3; // mmHg·s/m³
+        protected override double MillimeterMercurySecondsPerLiterInOnePascalSecondPerCubicMeter => 7.50063755419211e-6; // mmHg·s/l
+        protected override double MillimeterMercurySecondsPerMilliliterInOnePascalSecondPerCubicMeter => 7.5006375541921e-9; // mmHg·s/ml
+        protected override double PascalMinutesPerCubicCentimeterInOnePascalSecondPerCubicMeter => 1.66666666666667e-8; // Pa·min/cm³
+        protected override double PascalMinutesPerCubicMeterInOnePascalSecondPerCubicMeter => 1.66666666666667e-2; // Pa·min/m³
+        protected override double PascalMinutesPerLiterInOnePascalSecondPerCubicMeter => 1.66666666666667e-5; // Pa·min/l
+        protected override double PascalMinutesPerMilliliterInOnePascalSecondPerCubicMeter => 1.66666666666667e-8; // Pa·min/ml
+        protected override double PascalSecondsPerCubicCentimeterInOnePascalSecondPerCubicMeter => 1e-6; // Pa·s/cm³
+        protected override double PascalSecondsPerCubicMeterInOnePascalSecondPerCubicMeter => 1; // Pa·s/m³
+        protected override double PascalSecondsPerLiterInOnePascalSecondPerCubicMeter => 1e-3; // Pa·s/l
+        protected override double PascalSecondsPerMilliliterInOnePascalSecondPerCubicMeter => 1e-6; // Pa·s/ml
+        protected override double WoodUnitsInOnePascalSecondPerCubicMeter => 1.25010625903202e-7; // WU
```

Comment:
All the others are verified at
https://www.vcalc.com/wiki/Fluid-Resistance-Unit-Conversion